tests/worklist_server_tests.py

```diff
@@ -246,31 +246,6 @@ def test_RequireThat_MultipleWorklistRequests_ReturnsIdenticalWorklists_ByUseOfS
             self.assertEqual(worklist_item_one.ScheduledProcedureStepSequence[i].ScheduledProcedureStepDescription, worklist_item_two.ScheduledProcedureStepSequence[i].ScheduledProcedureStepDescription)
             self.assertEqual(worklist_item_one.ScheduledProcedureStepSequence[i].CommentsOnTheScheduledProcedureStep, worklist_item_two.ScheduledProcedureStepSequence[i].CommentsOnTheScheduledProcedureStep)
 
-    def test_RequireThat_Logfile_Returnslogfile_Withcontent(self):

```

Comment:
Here, you could change the tests so that they verifies that the seed file is not modified when reproduce is supplied, and that the file is modified if reproduce is not supplied.